packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/form-action/components/WorkflowEditActionForm.tsx

```diff
@@ -111,6 +124,26 @@ export const WorkflowEditActionForm = ({
     }
   };
 
+  const onFieldUpdate = (field: WorkflowFormActionField) => {
+    if (actionOptions.readonly === true) {
+      return;
+    }
+
+    const updatedFormData = formData.map((f) =>
+      f.id === field.id ? field : f,
+    );
```

Comment:
nitpick: I think we usually avoid single-character variables. `field` could be renamed to `updatedField.`